src/App.tsx

```diff
@@ -1,27 +1,74 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { useQueries } from '@tanstack/react-query';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+type SelectedGoods = 'allGoods' | 'firstFiveGoods' | 'allRedGoods';
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+export const App: React.FC = () => {
+  const [goodsSelect, setGoodsSelect] = useState<SelectedGoods>('allGoods');
+  const results = useQueries({
+    queries: [
+      { queryKey: ['allGoods'], queryFn: getAll, staleTime: 0 },
+      { queryKey: ['first5'], queryFn: get5First, staleTime: 0 },
+      { queryKey: ['allRed'], queryFn: getRedGoods, staleTime: 0 },
+    ],
+  });
```

Comment:
You should not make three requests at a time, it is better to make one request with one click on a button

https://user-images.githubusercontent.com/37452763/197801638-d26c58c2-e832-433d-a607-48d58e6170c6.mov